manager/manager.go

```diff
@@ -40,7 +40,7 @@ import (
 	"github.com/google/cadvisor/events"
 	"github.com/google/cadvisor/fs"
 	info "github.com/google/cadvisor/info/v1"
-	"github.com/google/cadvisor/info/v2"
+	v2 "github.com/google/cadvisor/info/v2"
```

Comment:
I don't think this does anything?

Reply:
same with container.go

Reply:
yes, it does nothing. Sorry for that it was just my editor autoformat on save.